astropy/table/tests/test_operations.py

```diff
@@ -661,7 +661,9 @@ def test_mixin_functionality(self, mixin_cols):
 
         # Check for left, right, outer join which requires masking. Works for
         # the listed mixins classes.
-        if isinstance(col, (Quantity, Time, TimeDelta, SkyCoord)):
+        if isinstance(
+            col, (Quantity, Time, TimeDelta, BaseRepresentationOrDifferential, SkyCoord)
```

Comment:
should we define a global for this tuple of classes at the top level of this module to avoid repetitions and possible future inconsistencies between tests ?

Reply:
Done!